lib/components/molecules/ArticleThumbnail/ArticleThumbnail.js

```diff
@@ -0,0 +1,66 @@
+// @flow
+/**
+ *
+ * ArticleThumbnail
+ *
+ */
+import React from 'react';
+import styled from 'styled-components';
+
+import styles from './ArticleThumbnail.style';
+import type { Props } from './types';
+import Image from '../../atoms/Image/Image';
+import Heading from '../../atoms/Heading/Heading';
+import Anchor from '../../atoms/Anchor';
+import Para from '../../atoms/Para';
+
+const getAnchoredChild = (url, children, target) => (
+  <Anchor to={url} target={target}>
+    {children}
+  </Anchor>
+);
+
+const getBaseVariation = (image, title, url, description, variation, readMoreText) => (
+  <React.Fragment>
+    {variation === 'v3' ? getAnchoredChild(url, <Image {...image} />) : <Image {...image} />}
```

Comment:
@bhardwajamit0890 I don't like the idea of this named variation, it should be your data driven. If you pass certain props the component will behave in certain way, that is much better way to handle it.